src/Configuration.hs

```diff
@@ -122,7 +131,10 @@ data Configuration = Configuration
     mergeWindowExemption :: MergeWindowExemptionConfiguration,
 
     -- Configuration for the Prometheus metrics server.
-    metricsConfig :: Maybe MetricsConfiguration
+    metricsConfig :: Maybe MetricsConfiguration,
+
+    -- Feature freeze period in which only 'merge hotfix' commands are allowed
+    featureFreezeWindow :: Maybe FeatureFreezeWindow
```

Comment:
Wouldn't it make sense to have this be a list of feature freeze windows? For our use cases, we have two, and they'll be the same every year. Would make sense to just configure it once and never touch them again.

Reply:
I don't think we have two, and they're definitely not on the same dates per year. My thinking was that there will only ever be one feature freeze active at a time, so there's no real need to be able to set multiple (and it's easier to implement this way).

Reply:
We do have two. But even so, having this as a list makes it easier to configure (no need to change the config between the two feature freezes) and the implementation isn't any more complicated.

Reply:
> We do have two

We don't. Also this is internal to the organization and we should probably not discuss this in a public repository.

> no need to change the config between the two feature freezes

We'll always have to update the configuration for the new feature freeze unless we plan two feature freezes ahead (we don't).

---

I'll be the person that will both do the implementation and the configuration, and I'm happy with the current implementation.